docs/service-introspection-onboarding.md

```diff
@@ -25,6 +45,115 @@ following properties of this storage before proceeding:
 Once you have a storage account with a table, you may proceed to start updating
 the pipelines to send data to Spektate storage.
 
+**Note:** The Azure storage account is needed to store information about your
+pipelines and services that is displayed by service introspection.
+
+2. Create a table. Follow these
+   [instructions](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal).
+
+### Pipelines Configuration
+
+The Bedrock GitOps pipelines need to be configured to start sending data to
+`spk` service introspection. This is done by adding a script snippet in each
+`azure-pipelines.yml` configuration.
+
+#### Configure a variable group
+
+To send data from Azure pipelines to the service introspection storage created
+previously a variable group needs to be configured in Azure DevOps (where the
+pipelines are).
+
+To configure the variable group run: `spk variable-group create`
```

Comment:
Adding a link to variable group command page helps here. Brief explanation also helps wrt variables which will need to be configured in the variable group. ACCOUNT_NAME: <storage account name from the above step> ACCOUNT_KEY: <storage account access key> TABLE_NAME: <table name from above step> PARTITION_KEY: <this can be any string>.

Reply:
there's no variable group command page, there's no documentation on that.

Reply:
We can add this link reference https://github.com/CatalystCode/spk/blob/master/docs/variable-group.md